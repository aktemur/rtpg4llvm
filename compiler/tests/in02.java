/*expr*/
 5.4  ;
