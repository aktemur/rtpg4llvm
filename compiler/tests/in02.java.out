mkConst(5.4)
